### ISSUE TITLE
[Bug]: RetrieveStatelessRates does not send CarrierAccounts correctly

### ISSUE THREAD

**Issue by EatonZ** [EasyPost/easypost-csharp#549]
### Software Version

6.2.0

### Language Version

.NET 8

### Operating System

Windows 11

### What happened?

In the `Client.Beta.Rate.RetrieveStatelessRates` method, you have the option to specify carrier account IDs. That can be done like this:

```
CarrierAccounts =
[
    new CarrierAccount
    {
        Id = CARRIER_ACCOUNT_ID_USPS
    },
    new CarrierAccount
    {
        Id = CARRIER_ACCOUNT_ID_UPSDAP
    }
]
```

However, they appear to be sent incorrectly and results in an error:

![Error](https://github.com/EasyPost/easypost-csharp/assets/3020941/396ab5db-c45c-4037-a9cc-9218230aebd8)

Reviewing the [API documentation](https://www.easypost.com/docs/api/csharp#retrieve-rates-for-a-shipment), I spotted the problem. The API expects an array of strings, not an array of objects. I have verified this by sending the request manually, and the error no longer occurs. Here is a correctly formatted request:

![Success](https://github.com/EasyPost/easypost-csharp/assets/3020941/6309e732-ccdb-4e0c-a2d7-20497c1afa63)

**Second issue:** Why, when sending carrier accounts, is it adding "Prefix=ca"? This seems unnecessary and should be removed.

### What was expected?

The API call should succeed when specifying carrier accounts.

**Comment by nwithan8** [comment]
Hello @EatonZ , thank you for submitting this issue, and for diving into the documentation to debug it.

You are correct, the API is in fact expecting a list of strings, great catch! This will be rectified in #550 and the subsequent release. You'll still set the `CarrierAccount` parameter as you were previously; behind the scenes, the library will convert the list of objects to a list of ID strings.

Thank you again, and thanks for testing out our beta endpoints!

**Comment by nwithan8** [comment]
This has been fixed in v6.2.1

**Comment by EatonZ** [comment]
@nwithan8 Thanks so much for the quick fix!
